client.go

```diff
@@ -87,11 +85,7 @@ func (c *Client) Rpc(name string, count string, rpcBody interface{}) string {
 	}
 
 	if count != "" && (count == `exact` || count == `planned` || count == `estimated`) {
-		if c.clientTransport.header.Get("Prefer") == "" {
```

Comment:
I don't think this stuff is necessary, since we initialize and execute the request within this method?

Reply:
I think it is necessary because rpc accepts this params according to [doc](https://postgrest.org/en/stable/api.html?highlight=exact#exact-count). Also user can define another Prefer in optional headers.

Reply:
Sorry, I was referring specifically to the merge of old and new values. I've just pushed a couple of bugfixes that should ensure that we don't need to do the merge here.